css/portal-dashboard/variables.less

```diff
@@ -25,6 +25,7 @@
 @cc-orange-light2: #ffc18a;
 @cc-orange-light3: #ffcea1;
 @cc-orange-light4: #ffe6d0;
+@cc-orange-light4B: #ffeddc;
```

Comment:
I'm sure it's fine as is, but I can't help but ask if making it `@cc-orange-light5` and then incrementing the next two variables' numbers accordingly would be problematic?

Reply:
Good question.  I tend to defer to Michael for color names and use whatever naming conventions that he uses in the UI spec.  This creates consistency when matching up colors in the UI spec and the actual application CSS.  In this case, Michael went with `cc-orange-light4B` for this new color in the UI spec so I used that.  If we changed the color names here, then they would no longer sync things up with the UI spec - I think that would be fairly confusing.

Reply:
OK, that seems very reasonable.